photon-lib/src/main/java/org/photonvision/PhotonCamera.java

```diff
@@ -55,11 +63,11 @@ public class PhotonCamera {
     DoublePublisher targetAreaEntry;
     DoubleArrayPublisher targetPoseEntry;
     DoublePublisher targetSkewEntry;
-    StringSubscriber versionEntry;
-    BooleanPublisher inputSaveImgEntry, outputSaveImgEntry;
-    IntegerEntry pipelineIndexEntry, ledModeEntry;
-    IntegerSubscriber heartbeatEntry;
 
+    /**
+     * Close out the NetworkTables entries used to communicate with the camera and releases their
+     * resources.
+     */
     public void close() {
```

Comment:
what is the point of this method, most teams create the object in RobotInit and then leave the object for the lifecycle of the robot code, this serves no purpose as far as I can tell.

Reply:
PhotonCamera should probably implement AutoCloseable, and we should make sure all the publishers/subscribers/entries are close()d. it's for NT JNI resource cleanup.

Reply:
Ok, I can add that then.